packages/truffle/test/scenarios/commands/build.js

```diff
@@ -0,0 +1,65 @@
+const assert = require("assert");
+const CommandRunner = require("../commandrunner");
+const MemoryLogger = require("../memorylogger");
```

Comment:
should these be commandRunner and memoryLogger for snake case? I'm not sure if there's a special reasons for capitalizing the first letter, haven't had my coffee yet!